### DISCUSSION TITLE
how do I skip a special rule by id for all tasks?

### DISCUSSION THREAD

**Discussion by braindevices** [ansible/ansible-lint#1694] [Q&A] [answered]
for example for E204, I tried `-x E204` or `-x 204`:
```
-x SKIP_LIST          only check rules whose id/tags do not match these values
```
Does not work at all.
I also tried .ansible-lint:
```
$ cat .ansible-lint 
---
skip_list:
  - '204'

```
does not help either, always show:

```
yaml: line too long (361 > 160 characters) (line-length)

```

**Comment by webknjaz** [comment, marked as answer]
Try
```console
$ ansible-lint -x line-length playbook.yml
```

Ref: https://ansible-lint.readthedocs.io/en/stable/rules.html#excluding-rules

**Reply by braindevices** [reply]
@webknjaz thanks this works! However, how do I find the correct rule name via the error number?

I try to use `-L`, but it only shows the tags

**Reply by webknjaz** [reply]
@ssbarnea may be able to comment on that

**Reply by andrewimeson** [reply]
for those who come back to this looking for answers about where to find the mapping of IDs to names:

https://github.com/ansible/ansible-lint/blob/v24.2.3/src/ansiblelint/constants.py#L96
